components/figure/Renderer.tsx

```diff
@@ -50,12 +50,20 @@ const Renderer: React.FC<RendererProps> = ({ document, rawDocument }) => {
     }
   }, []);
 
-  const handlePrint = useCallback(() => {
-    if (
-      navigator.userAgent.includes("SamsungBrowser") ||
-      navigator.userAgent.includes("CriOS") ||
-      navigator.userAgent.includes("FxiOS")
-    ) {
+  const handlePrint = useCallback(async () => {
+    const ua = window.navigator.userAgent;
+    const isIOS = /iPad|iPhone|iPod/.test(ua);
```

Comment:
Perhaps we can use the `i` flag for case **i**nsensitive:

```suggestion
    const isIOS = /iPad|iPhone|iPod/i.test(ua);
```

Reply:
Updated in https://github.com/Open-Attestation/verify.gov.sg/pull/170/commits/9d759ac09bde05891b6a5b0d4aec788f62a63b04